lib/manageiq/api/common/application_controller_mixins/exception_handling.rb

```diff
@@ -0,0 +1,37 @@
+module ManageIQ
+  module API
+    module Common
+      module ApplicationControllerMixins
+        module ExceptionHandling
+          def self.included(other)
+            other.rescue_from(StandardError, RuntimeError) do |exception|
+              errors, status = collected_ipp_errors(exception)
+
+              render :json => errors, :status => status
+            end
+          end
+
+          def collected_ipp_errors(exception, default_code = 400)
+            errors = []
+            og_exception = exception
```

Comment:
Not sure I understand the `og_` prefix either here.

Reply:
Shorthand for original, changed it to `top_level_exception`